Makefile

```diff
@@ -0,0 +1,53 @@
+## Copyright 2020 Red Hat, Inc.
+##
+## Licensed under the Apache License, Version 2.0 (the "License");
+## you may not use this file except in compliance with the License.
+## You may obtain a copy of the License at
+##
+##     http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+
+-include Makefile.overrides
+
+MAISTRA_VERSION ?= 1.1.0
+IMAGE           ?= docker.io/maistra/istio-ubi8-operator-metadata:${MAISTRA_VERSION}
```

Comment:
Should we have a `COMMUNITY` flag here like in `istio-operator`?

Reply:
This entire project is just community, hence no flag.